code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -197,14 +197,17 @@
 /datum/antagonist/traitor/proc/forge_traitor_objectives()
 	objectives.Cut()
 
+	//SKYRAT EDIT START - OBJECTIVE REMOVAL
+	/*
 	var/datum/objective/traitor_progression/final_objective = new /datum/objective/traitor_progression()
 	final_objective.owner = owner
 	objectives += final_objective
 
 	var/datum/objective/traitor_objectives/objective_completion = new /datum/objective/traitor_objectives()
```

Comment:
Won't this mean they don't get any objectives?

Reply:
They won't get the end-round objectives, as outlined in the PR